test/cypress/utils/verify-location.js

```diff
@@ -0,0 +1,9 @@
+import locations from '../../../lib/locations'
+
+export function verifyLocation(sdk, location) {
+  expect(sdk.location.is(location)).to.equal(true)
+  let invalidLocations = Object.values(locations)
+  invalidLocations = invalidLocations.filter(item => item !== location)
+  invalidLocations.push('test-value')
```

Comment:
Let's push `invalid-location` so it's more obvious

Reply:
fixed.